packages/changed-elements-react/src/api/VersionCompare.ts

```diff
@@ -33,6 +34,12 @@ export interface VersionCompareOptions {
    */
   changedElementsApiBaseUrl?: string | undefined;
 
+  /** Enable or disable display of side by side toggle in property comparison table */
+  displaySideBySideToggle?: boolean | undefined;
```

Comment:
Does this prop ever get passed down to the component that controls the side by side view?

Reply:
Good point i was thinking about it from the pineapple consumption point of view